Support/Scripts/run-lldb-tests.sh

```diff
@@ -125,6 +125,12 @@ else
   fi
 fi
 
+if [[ "${PLATFORM-}" = "1" ]]; then
```

Comment:
I think you should move this block to right before the final `exec` call, and use the strace stuff we have on lines 133-142. Probably do something like `ds2_path=...` (with the shell script or with just the binary), and then if you have `${PLATFORM-}`, just run $ds2_path "p" &, else `export LLDB_DEBUGSERVER_PATH="$ds2_path"`.